pkgs/development/tools/rust/cargo-web/default.nix

```diff
@@ -13,16 +22,34 @@ rustPlatform.buildRustPackage rec {
     sha256 = "1dl5brj5fnmxmwl130v36lvy4j64igdpdvjwmxw3jgg2c6r6b7cd";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-qualified-path.patch";
+      url = "https://github.com/koute/cargo-web/commit/c9584542163d60d0aae6d6890509794e838e257f.patch";
+      hash = "sha256-w59fXmrszptKt0llqGt0AF+0b3r9N6xUY6zQkpZnemE=";
+    })
```

Comment:
I'm not opposed to this fix but it make may sense to just disable warnings-as-error as clippy will keep getting more lints over time.